client/starwhale/api/_impl/experiment.py

```diff
@@ -0,0 +1,72 @@
+#  Copyright 2022 Starwhale, Inc. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import threading
+from typing import Any, Dict, Callable, Optional
+
+from starwhale.consts import DecoratorInjectAttr, DEFAULT_FINETUNE_JOB_NAME
+
+
+def fine_tune(*args: Any, **kw: Any) -> Any:
+    """
+    This function can be used as a decorator to define a fine-tune function.
+
+    Argument:
+        resources: [Dict, optional] Resources for the predict task, such as memory, gpu etc. Current only supports
+            the cloud instance.
+    Examples:
+    ```python
+    from starwhale import experiment
+
+    @experiment.fine_tune
```

Comment:
The line41-44 only supports `@experiment.fine_tune()`. If the users use `@experiment.fine_tune`, it may not work. 

```python
def fine_tune(*args: Any, **kw: Any) -> Any:
    if len(args) == 1 and len(kw) == 0 and callable(args[0]):
        return fine_tune()(args[0])
    else:
        def _wrap(func: Callable) -> Any:
            _register_ft(func, **kw)
            setattr(func, DecoratorInjectAttr.FineTune, True)
            return func
```
use the these code to support `@experiment.fine_tune` style?